### ISSUE TITLE
Congrats on retirement!

### ISSUE THREAD

**Issue by ying-w** [matloff/FarewellAddress#1]
It's almost been 2 decades since I took your classes at Davis, hope you enjoy your retirement!

**Comment by matloff** [comment]
Still trying to figure out who you are. :-)

**Comment by yihui** [comment]
> [...] hints should be doled out (sparingly) according to individual student needs, but giving detailed outlines in the assignment statement should be avoided

I agree with what you said [in the address](https://github.com/matloff/FarewellAddress/blob/master/Address.md#on-teaching). And

> People outside the field are amazed when I tell them that most CS students could not answer questions such as "How does a computer boot up?" and "What happens if we run a program with six threads on a four-core machine?"

I'm amazed when a CS professor could not figure out who a Github user is. Just kidding :) Here is a hint doled out sparingly by your "TA": https://github.com/ying-w/chipseq-compare

**Comment by matloff** [comment]
Thanks very much for the sparingly doled out hint. :-) but I seem to be in
need of even more hints.

How are you doing these days?

Norm
